health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -638,26 +748,26 @@ const createUpdatePlanProject = async (req) => {
-          delete updatedRow?.operatorName;
-          updatedRow.operator = state?.RuleConfigureOperators?.find((operation) => operation.operatorName === operatorName)?.operatorCode;
-          return updatedRow;
-        });
+        const formulasToUpdateFromUIForSubFormula = req?.formulasToUpdate
+          ?.filter((row) => {
+            return row.category && row.output && row.input && row.operatorName && row.assumptionValue;
+          })
+          ?.map((row) => {
+            const updatedRow = { ...row };
+            const operatorName = row?.operatorName;
+            delete updatedRow?.operatorName;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)